src/botPage/common/TicksService.js

```diff
@@ -200,6 +200,13 @@ export default class TicksService {
                 }
             }
 
+            if (data.msg_type === 'candles') {
+                const {
+                    subscription: { id },
+                } = data;
```

Comment:
@vinu-deriv Bhai just to be on the safe side can we add default values here so that it doesn't create runtime errors if any value is missing?
```suggestion
                const {
                  subscription: { id } = {},
              } = data || {};
```